site/gatsby-site/src/templates/cite.js

```diff
@@ -294,6 +299,27 @@ function CitePage(props) {
           parentIncident={incident}
         />
 
+        {spatialIncidents ? (
+          <>
+            <StyledHeading>
+              <Trans>This Incident in Semantic Space</Trans>
+            </StyledHeading>
+            <p>
+              <Trans>
+                The visualization below plots incidents closer together according to the similarity
+                of their reports texts, as identified through
+              </Trans>{' '}
+              <LocalizedLink to="/blog/using-ai-to-connect-ai-incidents">
+                <Trans>our natural-language processing system</Trans>
+              </LocalizedLink>
+              .
+            </p>
```

Comment:
This should be done in only one `Trans` component because the position of the link element might be different in other languages.

Reply:
La siguiente visualización representa los incidentes en proximidad proporcional a la similitud de los textos de sus informes, identificados a través de nuestro sistema de procesamiento de lenguaje natural.